prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -166,7 +166,7 @@ abstract class HistoryDatabaseAccess(
         orgService: String?,
         since: OffsetDateTime?,
         until: OffsetDateTime?,
-        showFailed: Boolean
+        showFailed: Boolean?
```

Comment:
Why not just set this to a default value like false?  Will be clearer I think.

Reply:
yeah that's a good point, updating...

Reply:
give the change a look!